kythe/java/com/google/devtools/kythe/extractors/java/bazel/JavaExtractor.java

```diff
@@ -162,7 +152,7 @@ public static void main(String[] args) throws IOException, ExtractionException {
                 javacOpts,
                 jInfo.getOutputjar());
 
-    if (outputFormat.equals(KZIP_OUTPUT_FORMAT)) {
+    if (outputPath.endsWith(IndexInfoUtils.KZIP_FILE_EXT)) {
```

Comment:
Can we flip this (i.e. if not .kzip, use .kindex) to ensure the legacy behavior of always using `.kindex` mostly remains until we delete support for it?

Reply:
Isn't that what this says, only with the positive case coming first?

Reply:
Carry on